### ISSUE TITLE
How to disable browser-sync? (to lose its Py/C++ dependencies on Windows)

### ISSUE THREAD

**Issue by barrystaes** [coryhouse/react-slingshot#42]
I'd rather not remove it entirely, thats destructive. Right now i dont actually use browser-sync, but at build it requires having Python and the Visual Studio C++ compiler installed on the Windows dev machine.

Is there a way to disable browser-sync at build to drop the Python / Visual Studio dependencies?

I did not have this problem until yesterday, when my Windows machine (that already had Python and VS installed for other reasons) decided to stop auto-magically working ..for no apparent reason. Apparently stuff on my Windows dev machine is colliding, and i wasted enough hours on that to take my loss and move on.

**Comment by coryhouse** [comment]
Hi Barry - Sorry to hear. Unfortunately, there's currently no way to simply disable it. You'd have to go into srcServer.js and simply utilize webpack's server without browser-sync's. 

We have a number of Windows machines at work using browser-sync just fine. Are you getting an error? Are you running Python 2.7? Might also check the browser-sync issues/forum to see if your issue is covered there.

**Comment by kwelch** [comment]
I didn't install python until just last week, but even prior to that I had no major issues with browser-sync.

**Comment by coryhouse** [comment]
Interesting. I thought it was browser-sync that required it, but it appears I was mistaken. [Their docs](https://www.browsersync.io/docs/#windows-users) only mention the C++ requirement on Windows, so your experience makes sense. It sounds like I can remove the Python requirement from the setup.

**Comment by coryhouse** [comment]
If you're looking for a way to host without browser-sync, I created a [simplified version of react-slingshot](https://github.com/coryhouse/ps-redux) that uses Webpack's dev server directly - so no browser-sync is required. I created it for my upcoming React with Redux Pluralsight course and it's still changing daily, but the Webpack dev server setup is complete and works.

So now that we have an example to reference for those interested in dumping browser-sync, closing.
